.eslintrc

```diff
@@ -7,6 +7,7 @@
     "import/external-module-folders": ["node_modules", ".yarn"]
   },
   "rules": {
+    "@typescript-eslint/no-unnecessary-type-assertion": "off",
```

Comment:
we often don't follow this rule, if you touch code that has type assertion you have to fix it, it is often hard to do without a risk